packages/shorebird_cli/lib/src/commands/upgrade_command.dart

```diff
@@ -83,8 +83,8 @@ class UpgradeCommand extends ShorebirdCommand {
       ['fetch', '--tags'],
       workingDirectory: workingDirectory,
     );
-    // Get the latest commit revision of the upstream
-    return _gitRevParse('@{upstream}', workingDirectory: workingDirectory);
+    // Get the latest commit revision of the stable branch
+    return _gitRevParse('origin/stable', workingDirectory: workingDirectory);
```

Comment:
This won't be right once we have channels.  Are you sure you want this?

Reply:
It's currently breaking because `@{upstream}` doesn't point to a branch

Reply:
Go for it then.  We can always change it later. :)